Cargo.toml

```diff
@@ -14,7 +14,7 @@ indicatif = "0.11"
 itertools = "0.6"
 log = "0.4.6"
 fern = "0.5.7"
-rust-htslib = "0.15"
+rust-htslib = {git="https://github.com/rust-bio/rust-htslib"}
```

Comment:
Right now this points to the master branch of rust-htslib. As soon as the next release of rust-htslib is out, this can be changed back, right? Is there a good way to not forget that? Should we make make an issue for that?

Reply:
> Just to document: @FelixMoelder noticed, that the access to the `HeaderView` does not really need a cloning of the header, but can be accessed through the record itself. However, that leads to mutable / immutable borrow clashes at some points, where the standard `rust-htslib` functions to get record fields (`format()`, `info()`, `genotype()`) require a mutable reference to `self` by default. We think these default functions should only require a regular (immutable) reference by default, and mutable access should be given by separate functions (`format_mut()`, `info_mut()`, `genotype_mut()`). So @FelixMoelder will:
> 
>     * [ ]  make this API breaking change over in the current rust-htslib master branch
> 
>     * [ ]  add the info about the API breaking change to the [rust-htslib changelog](https://github.com/rust-bio/rust-htslib/blob/master/CHANGELOG.md)

Looking at the code of rust-htslib shows that solving the immutable borrow clashes is not just straight forward.
While `genotype()` does not need a mutable reference, changing `info()` is more complex as it returns a `Info` object which also requires a mutable reference of the record. So we would need to revise the whole `Info` struct. For this reason we should discuss the further proceeding in detail before making any changes.

Reply:
> Right now this points to the master branch of rust-htslib. As soon as the next release of rust-htslib is out, this can be changed back, right? Is there a good way to not forget that? Should we make make an issue for that?

Yes, this should be changed as soon as there is a new release of rust-htslib. Currently the `bindgen` dependency conflicting with `rocksdb` is only solved in the master branch.